SwiftMonkey/MonkeyXCTest.swift

```diff
@@ -34,7 +34,7 @@ extension Monkey {
                 let alert = application.alerts.element(boundBy: i)
                 let buttons = alert.descendants(matching: .button)
                 XCTAssertNotEqual(buttons.count, 0, "No buttons in alert")
-                let index = UInt(self!.r.randomUInt32() % UInt32(buttons.count))
+                let index = Int(self!.r.randomUInt32() % UInt32(buttons.count))
```

Comment:
Could you use `randomInt()` method here? The code would look like this:
```swift
let index = self!.r.randomInt(lessThan: buttons.count)
```

Credits to @mohamede1945 who used this in his PR (#47).